apps/native/app/src/ui/lib/text-field/text-field.tsx

```diff
@@ -35,7 +35,7 @@ const Label = styled.Text`
   margin-bottom: 4px;
 `
 
-const Input = styled.TextInput`
+const Input = styled(TextInput)`
```

Comment:
This was causing TS error after style components update. The main reason for the styled-components update was inconsistent types between React, React native and styled-components.